flavors/integration_test/flavor_base/build_steps.py

```diff
@@ -0,0 +1,54 @@
+from typing import Dict
+
+from exasol.slc.internal.tasks.build.docker_flavor_image_task import (
+    DockerFlavorAnalyzeImageTask,
+)
+
+
+class AnalyzeBaseTestBuildRun(DockerFlavorAnalyzeImageTask):
+
+    def get_build_step(self) -> str:
+        return "base_test_build_run"
+
+    def requires_tasks(self):
+        return {}
+
+    def get_additional_build_directories_mapping(self) -> Dict[str, str]:
+        return {}
+
+    def get_path_in_flavor(self):
+        return "flavor_base"
+
+
+class AnalyzeFlavorTestBuildRun(DockerFlavorAnalyzeImageTask):
```

Comment:
Do we need this?

Reply:
This is the goal for the linker namespace tests. So I think it would be good to keep it here....

Reply:
No the goal for the linker namespace is AnalyzeBaseTestBuildRun, I think.

Reply:
I see. Removed it